packages/playwright-core/src/server/trace/recorder/tracing.ts

```diff
@@ -45,7 +45,7 @@ import type { ConsoleMessage } from '../../console';
 import { Dispatcher } from '../../dispatchers/dispatcher';
 import { serializeError } from '../../errors';
 
-const version: trace.VERSION = 6;
+const version: trace.VERSION = 7;
```

Comment:
There is no need to bump the version unless you have a meaningful `modernize_6_to_7` method. We can auto-detect by the presence of `stepId` in one of the actions.